mitre/system/ksp-persistence_boot_or_logon_IS_rc_script.yaml

```diff
@@ -10,7 +10,6 @@ metadata:
 spec:
   tags: ["MITRE","system","K8s","T1037"]
   message: "Warning! someone accessed sensitive process or files"
```

Comment:
This is good catch, I can merge this if you'll raise as separate PR, thanks for pointing this out @afzalbin64